### DISCUSSION TITLE
Operator Mono not recognized as a monospaced font

### DISCUSSION THREAD

**Discussion by andremedeiros** [blinksh/blink#1580] [Support]
I have Operator Mono, patched with ligatures, installed on iOS through Fontcase. Fontcase recognizes this as a monospaced font. Blink, however, does not recognize the font (nor does it show it on the font list.)

**Comment by markpanado** [comment]
You can try to convert that font into CSS (like this https://github.com/blinksh/fonts/blob/master/Hack%20Nerd%20Font.css) and manually import it in Blink
